middleware/modelExport.js

```diff
@@ -261,11 +261,11 @@ module.exports = {
             req.linz.export = getExport(list.export);
             req.linz.export.fields = {};
 
-            // retrieve the form to provide a list of fields to choose from
-            req.linz.model.getForm(req, function (formErr, form){
+            // retrieve the labels to provide a list of fields to choose from
+            req.linz.model.getLabels((labelErr, labels) => {
```

Comment:
@allanchau, should we update this to use the model schema rather than the labels? The labels might not be complete either...

Reply:
In a previous update, the defaults were added to all the labels.
If we used the schema, I'm not sure how we could grab the label?

Reply:
Right so the purpose here is to grab the labels, rather than a list of all fields?

Reply:
Sort of, it is used to display the fields in the popup and as the csv headers.
`L63 /lib/formtools/plugins/document.js` seems to provide complete coverage for all fields besides _id?